.github/workflows/ci_cd_pr.yml

```
@@ -98,7 +100,7 @@ jobs:
     needs: changelog-fragment
     steps:
       - name: "Run code style checks"
-        uses: ansys/actions/code-style@main
+        uses: ansys/actions/code-style@feat/check-actions-security-action
```

```suggestion
        uses: ansys/actions/code-style@main
```
Needs to be reverted before merging